dpkg/pushtopandas.py

```diff
@@ -14,8 +14,9 @@ def push_to_pandas(directory, object_id_cmso):
     storage = dp.push_datapackage(descriptor=descr, backend='pandas')
     print(storage.buckets)
 
-    objects = storage['objects___objectstable']
-    links = storage['links___linkstable']
+    # FIXME: the following is hardwired
```

Comment:
I have looked more deeply into this issue. The `push_datapackage` function generates those keys based on the resource's path and name:

```python
    table = os.path.splitext(path)[0]
    table = table.replace(os.path.sep, '__')
    if name is not None:
        table = '___'.join([table, name])
    table = re.sub('[^0-9a-zA-Z_]+', '_', table)
    table = table.lower()
    return table
```

Until datapackage version 0.x this was implemented in the public API (`from datapackage.mappers import convert_path`), but they have moved it to an internal function in` datapackage.pushpull` for the upcoming 1.0.0 version (they are at v1.0.0-alpha4 now). This is not good. I will open an issue on their repo. We can implement a workaround in the meantime.